Saga methods marked async but don't await anything

Results in a compiler warning:

```
warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
```

Consider marking https://github.com/Particular/Workshop.Microservices/blob/master/exercises/03%20-%20Sagas.md#step-4-2 to not be async and return a Task:

```c#
    private Task ProcessOrder(IMessageHandlerContext context)
    {
        if (Data.IsOrderSubmitted && Data.IsPaymentProcessedYet)
        {
            // Send a message to execute shipment
            MarkAsComplete();
        }
        return Task.CompletedTask;
    }
```

--- a/exercises/03 - Sagas.md
+++ b/exercises/03 - Sagas.md
@@ -220,7 +220,7 @@ In the saga add a new async method called `ProcessOrder` that you will call from
     {
         if (Data.IsOrderSubmitted && Data.IsPaymentProcessedYet)
         {
-            // Send a message to execute shipment
+            await Task.CompletedTask; // Send a message to execute shipment
             MarkAsComplete();
         }
     }

--- a/exercises/src/03 Sagas/after/Divergent.Shipping/Sagas/ShippingSaga.cs
+++ b/exercises/src/03 Sagas/after/Divergent.Shipping/Sagas/ShippingSaga.cs
@@ -19,7 +19,7 @@ namespace Divergent.Shipping.Sagas
             mapper.ConfigureMapping<PaymentSucceededEvent>(p => p.OrderId).ToSaga(s => s.OrderId);
         }
 
-        public async Task Handle(OrderSubmittedEvent message, IMessageHandlerContext context)
+        public Task Handle(OrderSubmittedEvent message, IMessageHandlerContext context)
         {
             Log.Info("Handle OrderSubmittedEvent");
 
@@ -30,24 +30,24 @@ namespace Divergent.Shipping.Sagas
             var projection = message.Products.Select(p => new ShippingSagaData.Product { Identifier = p });
             Data.Products = projection.ToList();
 
-            await ProcessOrder(context);
+            return ProcessOrder(context);
         }
 
-        public async Task Handle(PaymentSucceededEvent message, IMessageHandlerContext context)
+        public Task Handle(PaymentSucceededEvent message, IMessageHandlerContext context)
         {
             Log.Info("Handle PaymentSucceededEvent");
 
             Data.OrderId = message.OrderId;
             Data.IsPaymentProcessedYet = true;
 
-            await ProcessOrder(context);
+            return ProcessOrder(context);
         }
 
         private async Task ProcessOrder(IMessageHandlerContext context)
         {
             if (Data.IsOrderSubmitted && Data.IsPaymentProcessedYet)
             {
-                // Send a message to execute shipment
+                await Task.CompletedTask; // Send a message to execute shipment
                 MarkAsComplete();
             }
         }

